lisa/notifiers/junit.py

```diff
@@ -300,6 +300,18 @@ def _add_test_case_result(
         testcase.attrib["classname"] = class_name
         testcase.attrib["time"] = self._get_elapsed_str(elapsed)
 
+        # Add a standard system-properties element for all test cases
+        if hasattr(message, "information") and isinstance(message.information, dict):
```

Comment:
Please make sure it works within Azure DevOps test results. A few teams tried to add it, but found it either not working or breaking the test results. Please make sure no regression on ADO test results.